portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,22 +40,29 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    ArrayList<String> jsonString = new ArrayList<>();
-    addToList();
-    for(String obj : strings){
-        jsonString.add(convertToJSON(obj));
+
+    Query query = new Query("Comments").addSort("comment", SortDirection.DESCENDING);
```

Comment:
WIll this sort comments alphabetically?

Reply:
Sorry I've done some touch up after the pull request. In this commit I have updated the comments to be displayed based on the time it's been submitted https://github.com/Phoebe-Khokgawe/STEP_Starter/pull/9/commits/f74d300f381d94330b947b5ff24e9ef63405083f.